src/hooks/api/usePatchUserInfo.ts

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { useMutation } from 'react-query';
+import { patchUserInfo } from '@/api';
+import { SurveyFields } from '@/types';
+
+const usePatchUserInfo = () => {
+  const [isPatchSuccess, setIsPatchSuccess] = useState(false);
+  const { mutate, isLoading, error, isError } = useMutation(
+    (surveyState: SurveyFields) => patchUserInfo(surveyState),
+    {
+      onSuccess: () => {
+        // TODO: 안드팀에 토큰 보내주기
+        setIsPatchSuccess(!isLoading && !isError);
+      },
+      onError: (Error) => {
+        // TODO: 에러화면 보여주기
+        console.log(Error);
+      },
+    }
+  );
+  return {
+    mutate,
+    isLoading,
+    error,
+    isError,
+    isPatchSuccess,
```

Comment:
```ts
  const [isPatchSuccess, setIsPatchSuccess] = useState(false);
```
위의 7 line을 아예 없애버리고
26 line에 아래처럼 해서 isPatchSuccess를 만들면 필요없는 useState를 하나 제거할 수 있을 것 같은데 이런 방식은 어떠실까요?
```suggestion
    isPatchSuccess: !isLoading && !isError,
```